src/minmax.rs

```diff
@@ -46,11 +46,10 @@ impl<T: Clone> MinMaxResult<T> {
 }
 
 /// Implementation guts for `minmax` and `minmax_by_key`.
-pub fn minmax_impl<I: Itertools, K, F, L, G>(mut it: I, mut key_for: F,
-                                             lt: L, gt: G) -> MinMaxResult<I::Item>
+pub fn minmax_impl<I: Itertools, K, F, L>(mut it: I, mut key_for: F,
+                                             mut lt: L) -> MinMaxResult<I::Item>
```

Comment:
Great PR. Indent/align mismatch here because of the G being removed above. The `I` type param should really be bounded by Iterator, not itertools. (And I: Sized is redundant).